cmdLine/src/main/scala/topgun/cmdline/FileParser.scala

```diff
@@ -154,13 +155,25 @@ class FileParser(file: File, cmdLine: JfrParseCommandLine, totals: Totals, confi
 
   private def readFrames(event: RecordedEvent): List[CallSite] = {
     val stack = event.getStackTrace
-
+    val isNative = if(event.getEventType.getLabel=="Method Profiling Sample Native") true else false
```

Comment:
`if` is not needed
`val isNative = event.getEventType.getLabel=="Method Profiling Sample Native") `